inductiva/molecules/scenarios/protein_solvation/protein_solvation.py

```diff
@@ -43,52 +43,27 @@ def __init__(
     def simulate(
             self,
             simulator: Simulator = GROMACS(),
-            working_dir: Optional[Path] = None,
+            output_dir: Optional[Path] = None,
             simulation_time: float = 10,  # ns
-            integrator: str = "md",
+            integrator: Literal["md", "sd", "bd"] = "md",
```

Comment:
I am still in favour of this being a bit more descriptive, since md/sd/bd don't bring anything to mind, unless you are a gromacs expert.